moshi/src/test/java/com/squareup/moshi/JsonQualifiersTest.java

```diff
@@ -339,9 +339,6 @@
       assertThat(expected).hasCauseExactlyInstanceOf(IllegalArgumentException.class);
       assertThat(expected.getCause()).hasMessage("No @FromJson adapter for class java.lang.String "
           + "annotated [@com.squareup.moshi.JsonQualifiersTest$FooPrefix()]");
-      assertThat(expected).hasCauseExactlyInstanceOf(IllegalArgumentException.class);
-      assertThat(expected.getCause()).hasMessage("No next JsonAdapter for class "
-          + "java.lang.String annotated [@com.squareup.moshi.JsonQualifiersTest$FooPrefix()]");
```

Comment:
i think we still want the tests from #659

the two statement above should be
```java
assertThat(expected.getCause()).hasCauseExactlyInstanceOf(IllegalArgumentException.class);
assertThat(expected.getCause().getCause()).hasMessage("No next JsonAdapter for class "
    + "java.lang.String annotated [@com.squareup.moshi.JsonQualifiersTest$FooPrefix()]");
```

Reply:
I see, cool. fixed it so.